### PR TITLE
Added bombermaaan.sh script.

### PR DESCRIPTION
**Pull request by bjaraujo**
I would like if you could add this port to retropie experimental for others to install and test.

### CODE REVIEW

File: scriptmodules/ports/bombermaaan.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+function sources_bombermaaan() {
+    gitPullOrClone "$md_build" https://github.com/bjaraujo/Bombermaaan.git
+}
+
+function build_bombermaaan() {
+    cd trunk
+    cmake . -DCMAKE_INSTALL_PREFIX="$md_inst" -DLOAD_RESOURCES_FROM_FILES:BOOL=ON
+    make
+    mv src/Bombermaaan bombermaaan
```
Comment (joolswills) [inline scriptmodules/ports/bombermaaan.sh]: I don't think the mv is needed. Can just check for the file in trunk/src

File: scriptmodules/ports/bombermaaan.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        'trunk/res/images'
+        'trunk/res/sounds'
+    )
+}
+
+function configure_bombermaaan() {
+    mkUserDir "$home/.config"
+    moveConfigDir "$home/.config/bombermaaan" "$md_conf_root/bombermaaan"
+    addPort "$md_id" "bombermaaan" "Bombermaaan" "pushd $md_inst; $md_inst/bombermaaan; popd"
+}
```
Comment (joolswills) [inline scriptmodules/ports/bombermaaan.sh]: missing EOL

File: scriptmodules/ports/bombermaaan.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        'trunk/bombermaaan'
+        'trunk/levels'
+        'trunk/res/images'
+        'trunk/res/sounds'
+    )
+}
+
+function configure_bombermaaan() {
+    mkUserDir "$home/.config"
+    moveConfigDir "$home/.config/bombermaaan" "$md_conf_root/bombermaaan"
```
Comment (joolswills) [inline scriptmodules/ports/bombermaaan.sh]: Just want to confirm it actually uses this location for config storage ?

**Comment by joolswills** [comment]
Made a few more comments. I haven't had a chance to test yet (which I usually do before merging)

File: scriptmodules/ports/bombermaaan.sh (hunk 1 above)
Comment (bjaraujo) [reply scriptmodules/ports/bombermaaan.sh]: mv is just to move and rename to lower case.

File: scriptmodules/ports/bombermaaan.sh (hunk 3 above)
Comment (bjaraujo) [reply scriptmodules/ports/bombermaaan.sh]: It probably shouldn't need that. Should I delete that?

File: scriptmodules/ports/bombermaaan.sh (hunk 2 above)
Comment (bjaraujo) [reply scriptmodules/ports/bombermaaan.sh]: I will add.

File: scriptmodules/ports/bombermaaan.sh (hunk 3 above)
Comment (joolswills) [reply scriptmodules/ports/bombermaaan.sh]: Yes please.

File: scriptmodules/ports/bombermaaan.sh (hunk 1 above)
Comment (joolswills) [reply scriptmodules/ports/bombermaaan.sh]: ok. I'm happy with that.

**Comment by bjaraujo** [comment]
Ok. I have made the changes. Please test.

**Comment by joolswills** [comment]
Thanks.

**Pull request merged**